mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/76 - game status update

Closes #76 

To test:
- Run scenario
- Make sure current campaign, current scenario, and simulation date (month and day) show correctly on the Main UI

Notes:
- Still need to display the current zone(?) the user is looking at
![image](https://user-images.githubusercontent.com/41123045/105519037-d2b2a800-5c8d-11eb-8bf0-d3241765ef3e.png)
